src/docfx/build/Build.cs

```diff
@@ -38,17 +38,25 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Erro
                 (Docset docset, Docset fallbackDocset) GetDocsetWithFallback()
```

Comment:
try turn inner function into member function first

Reply:
Turned inner function `GetDocsetWithFallback()` and `ShouldBuildFile()` to member function, and resolved the line too long issue.